### ISSUE TITLE
Links on menu-bar don't link anywhere

### ISSUE THREAD

**Issue by abisummers** [sudz123/Happy_Hacking#235]
on menu-bar.html there are <a> tags but the links don't work

**Comment by sudz123** [comment]
I think someone had made the menu bar but it then all the code was shifted to src. Maybe you can move this and the other code files which are outside. However I don't think there is anyway we can reach this page through home.html

**Comment by abisummers** [comment]
@sudz123  I've moved everything into the src file. Is there a need for the menu-bar.html? It seems that everything there is already included on the home page. The youtube video also doesn't appear to work on the menu-bar page

**Comment by sudz123** [comment]
Maybe you can put this to good use and add a page of your own, use the code in the menu bar.html and add a link in the footer of home.html to your own html page? This repo is for beginners and so there is a lot of redundant code indeed.

**Comment by abisummers** [comment]
Yeah, I understand it's for beginners, which is one of the many reasons I love hacktoberfest (and specifically this project🎉). I had no intentions of deleting the file, but it's good to know that it can be changed and that it's open for anyone to add their own twist on it. 

However, I do think the css (menu-bar.css) is pretty hard  to follow for beginners, so maybe it's worth removing some of it to make it more beginner friendly. A lot of the css is not used and if someone then wanted to style their page, the css may interfere

**Comment by manishreddy6305** [comment]
I would like to work on this issue. Can you assign this task to me ?
